packages/zoe/test/jsconfig.json

```diff
@@ -0,0 +1,18 @@
+// This file can contain .js-specific Typescript compiler config.
```

Comment:
Are we consolidating these files across our packages? Do the settings here differ from other packages?

Reply:
@michaelfig added this, I'm not sure

Reply:
This file was only necessary to turn on bigint support in the tests without requiring them to be typechecked on lint.